test/e2e/page-objects/pages/bridge/quote-page.ts

```diff
@@ -62,12 +62,21 @@ class BridgeQuotePage {
     // Destination
     await this.driver.waitForSelector(this.destinationAssetPickerButton);
     await this.driver.clickElement(this.destinationAssetPickerButton);
-    await this.driver.clickElement(`[data-testid="${quote.toChain}"]`);
+    await this.driver.clickElementAndWaitToDisappear(
+      `[data-testid="${quote.toChain}"]`,
+    );
     await this.driver.fill(this.assetPrickerSearchInput, quote.tokenTo);
-    await this.driver.clickElement({
+    await this.driver.clickElementAndWaitToDisappear({
       text: quote.tokenTo,
       css: this.tokenButton,
     });
+    await this.driver.assertElementNotPresent(
+      {
+        tag: 'p',
+        text: 'Fetching quotes...',
+      },
+      { waitAtLeastGuard: 5000 },
+    );
```

Comment:
the 3 changes above fix the race condition where we couldn't find the expected token/amount in the tx list, because the UI was not taking the custom values we set, and then the tx activity showed miss-matching amounts or tokens